modules/test/playwright/pages/object-web/model-builder/ModelBuilderObjectDefinitionNodePage.ts

```diff
@@ -51,6 +52,10 @@ export class ModelBuilderObjectDefinitionNodePage {
 		this.editObjectDefinitionExternalReferenceCodeButton = page
 			.getByText('Edit ERC')
 			.last();
+		this.modalAddObjectField = page.locator('form#modalAddObjectField');
+		this.modalAddObjectFieldLabelInput = this.modalAddObjectField.locator(
+			'input[name="label"]'
+		);
```

Comment:
Hi @larissacribeiro, I imagine you've already tried the most recommended locators. Could you share what types of errors you encountered?